base/shell/explorer/taskband.cpp

```diff
@@ -286,37 +288,18 @@ class CTaskBand :
 
     /*****************************************************************************/
 
-    virtual HRESULT STDMETHODCALLTYPE ProcessMessage(
-        IN HWND hWnd,
-        IN UINT uMsg,
-        IN WPARAM wParam,
-        IN LPARAM lParam,
-        OUT LRESULT *plrResult)
-    {
-        TRACE("CTaskBand: IWinEventHandler::ProcessMessage(0x%p, 0x%x, 0x%p, 0x%p, 0x%p)\n", hWnd, uMsg, wParam, lParam, plrResult);
-        return E_NOTIMPL;
-    }
-
-    virtual HRESULT STDMETHODCALLTYPE ContainsWindow(
-        IN HWND hWnd)
```

Comment:
What have these 2 functions been removed?

Reply:
Maybe it's from some past inheritance of CTaskBand to something like CISFBand or whatnot:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=ContainsWindow